domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/domene/iay/InntektArbeidYtelseGrunnlag.java

```diff
@@ -196,7 +196,10 @@ public Optional<OppgittOpptjening> getOverstyrtOppgittOpptjening() {
      * Returnerer overstyrt oppgitt opptjening hvis det finnes, eller vanlig oppgitt opptjening
      */
     public Optional<OppgittOpptjening> getGjeldendeOppgittOpptjening() {
-        return Optional.ofNullable(getOverstyrtOppgittOpptjening()).orElseGet(this::getOppgittOpptjening);
+        if (getOverstyrtOppgittOpptjening().isPresent()) {
```

Comment:
Kan du ikke bare beholde denne så vi unngår flere kall til optexpr?
return getOverstyrtOppgittOpptjening().orElseGet(this::getOppgittOpptjening);